snc.sh

```diff
@@ -239,11 +239,15 @@ retry ${OC} delete mc chronyd-mask
 # Wait for the cluster again to become stable because of all the patches/changes
 wait_till_cluster_stable
 
+mc_before_removing_pullsecret=$(retry ${OC} get mc --sort-by=.metadata.creationTimestamp --no-headers -oname)
 # Replace pull secret with a null json string '{}'
 retry ${OC} replace -f pull-secret.yaml
+mc_after_removing_pullsecret=$(retry ${OC} get mc --sort-by=.metadata.creationTimestamp --no-headers -oname)
 
-# Wait for the cluster again to become stable because of remove of pull secret
-wait_till_cluster_stable
+while [ "${mc_before_removing_pullsecret}" == "${mc_after_removing_pullsecret}" ]; do
```

Comment:
Should these lists be filtered on `rendered_worker` or `rendered_master` in case an unrelated machineconfig comes up or disappears?

Reply:
@cfergeau ideally if we are not making any change in the mco then this shouldn't happen and if that is the case then we need to file bug on mco side.

Reply:
Dunno, sounds like relying a bit too much on something we don't control :)

Reply:
@cfergeau As soon as we hit `openshift-installer` we really don't control the flow till cluster is provisioned so if there is issue it should be openshift side.